Makefile

```diff
@@ -25,18 +33,21 @@ all: dev
 dev: build simulator check test
 
 build:
+	go get -u github.com/golang/dep/cmd/dep
+	dep ensure
+	dep prune
```

Comment:
I think it's better to move them to 'make update'.

Reply:
+1